### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by deliciousturtle**
### Author
Khang Nguyen (43129681)

### Problem
Classify laterality (left or right sided knee) of the OAI AKOA knee data set having a minimum accuracy of 0.9 on the test set.

### Description
Includes 2 script files, a README.md and 2 pngs (plots).

`classify_laterality.py` is the driver script and is used via the cmd line by

```
>python classify_laterality.py <path_to_data_folder>
```
where `<path_to_data_folder>` is the path to the OAI AKOA dataset folder. The driver script loads and preprocesses the data, builds the model, trains the model and then verifies and plots the results. 

`cnn.py` is the CNN module that is used in `classify_laterality.py` to build.

Please refer to the README.md for more detailed information.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Easy Difficulty 
- Results are good 

Other Feedback
- I would break up that 200 line function into smaller self-contained functions
- Good grouping of png files

**Comment by rexsw** [comment]
Code looks good for easy difficulty

Good use of tensorflow 

Be sure to delete any commented out code that is no longer needed 

Comments such as:
      #Compile the model
      model.compile(
Don’t add to the readability of the code and should be left out. But overall well commented.

File: recognition/laterality_cnn/classify_laterality.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
+import matplotlib.pyplot as plt
```
Comment (shakes76) [inline recognition/laterality_cnn/classify_laterality.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
